sqlglot/dialects/postgres.py

```diff
@@ -71,6 +72,7 @@ class Tokenizer(Tokenizer):
         }
 
     class Parser(Parser):
+        NULL_ORDERING = NullOrdering.NULLS_ARE_LARGE
```

Comment:
for parameters that are done both parser and generator, you can make it a lower case class instance,

null_ordering = "large"